src/themes/clean/templates/journal/article.html

```diff
@@ -118,7 +121,6 @@ <h2>{% trans "Abstract" %}</h2>
                         {% endif %}
```

Comment:
This is no longer inside the if galleys check.

Reply:
ta, its been a bit of a monster to find all the relevant bits, so I'm glad you spotted that.